### PR TITLE
Add configuration documentation for Service Log integration

### PR DESCRIPTION
**Pull request by JiriPapousek**
# Description

Documentation for configuration relevant to Service Log

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by tisnik — APPROVED** [review]
LGTM

**Review by epapbak — APPROVED** [review]

File: docs/configuration.md [hunk 1]
```diff
@@ -50,3 +50,48 @@ via `AGG_CONFIG` environment variable:
...
+- `token_refreshment_url` is a URL of OpenID Connect API for retrieval of new online token
+- `timeout` is a time used as a timeout when sending requests to Service Log API
+- `likelihood_threshold`,`impact_threshold`, `severity_threshold` and `total_risk_threshold` are values which can be used in `event_filter` for filtering messages sent to Service Log
+- `event_filter` is a condition string used to determine which messages will be sent to Service Log
+
+Please note that for correct functionality of Service Log integration, `dependencies` configuration should be also present.
+
+## Dependencies configuration
+
+Service Log configuration is in section `[dependencies]` in config file
```
Comment (juandspy) [inline docs/configuration.md:83]: is this sentence ok? this doesn't contain the SL configuration

**Review by juandspy — APPROVED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (JiriPapousek) [reply docs/configuration.md:83]: You are right, sorry. Will fix in a moment.

Comment (JiriPapousek) [reply docs/configuration.md]: Fixed

**Pull request merged**